make-scripts/start_bridge.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+if ! command -v npm &> /dev/null
+then
+    echo "npm could not be found, install npm first"
+    exit
+fi
+
+cd ./bridge/
+
+# install the dependencies
+npm install
+
+# accept the API URL and API TOKEN as the user inputs
+read -p 'Enter API URL: ' API_URL
```

Comment:
Can you make it such that if `API_URL` is already set, you just print it. Else, you use `read ...`

Reply:
sure 👍 
Well is it a good idea to print the API_TOKEN? just displaying that it exists maybe.

Reply:
You're right, just print an info "Using API_TOKEN environment variable" without printing the actual value.